app/Repositories/Summit/DoctrineSummitAttendeeBadgeRepository.php

```diff
@@ -107,4 +111,55 @@ public function getBadgeByTicketNumber(string $ticket_number): ?SummitAttendeeBa
 
         return $query->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * @param int $summit_id
+     * @param PagingInfo $paging_info
+     * @param Filter|null $filter
+     * @param Order|null $order
+     * @return PagingResponse
+     */
+    public function getBadgesBySummit(int $summit_id,
+                                      PagingInfo $paging_info,
+                                      Filter $filter = null,
+                                      Order $order = null): PagingResponse
+    {
+        $query  = $this->getEntityManager()
+            ->createQueryBuilder()
+            ->select("e")
+            ->from($this->getBaseEntity(), "e")
+            ->join("e.ticket", "t")
```

Comment:
ticket should be on paid status and active

Reply:
Even though the ticket status doesn´t matter when we are regenerating QR codes, we could manage this using the Filter param